platform/src/main/java/org/stellar/anchor/platform/rpc/NotifyCustomerInfoUpdatedHandler.java

```diff
@@ -59,13 +70,31 @@ public RpcMethod getRpcMethod() {
 
   @Override
   protected SepTransactionStatus getNextStatus(
-      JdbcSepTransaction txn, NotifyCustomerInfoUpdatedRequest request)
-      throws InvalidRequestException {
+      JdbcSepTransaction txn, NotifyCustomerInfoUpdatedRequest request) throws AnchorException {
+    String status = null;
+    if (request.getCustomerId() != null) {
```

Comment:
We should probably require the business to provide the customer ID and type in 3.0 right?

Reply:
Yeah